tools/meteor-services/config.js

```diff
@@ -102,8 +102,13 @@ export function getLocalPackageCacheFilename(serverUrl) {
 export function getPackageStorage(options) {
   options = options || {};
   var root = options.root || tropohouse.default.root;
-  return pathJoin(root, "package-metadata", "v2.0.1",
-    getLocalPackageCacheFilename(options.serverUrl));
+  if (process.env.METEOR_WAREHOUSE_METADATA_DIR) {
+    return pathJoin(process.env.METEOR_WAREHOUSE_METADATA_DIR, "v2.0.1",
```

Comment:
Would be a bit nicer if we store metadata dir in a variable first (branching on `process.env.METEOR_WAREHOUSE_METADATA_DIR`), so we don't need to duplicate the common parts: `"v2.0.1", getLocalPackageCacheFilename(options.serverUrl)`.

Reply:
Fixed